packages/decoder/test/current/test/downgrade-test.js

```diff
@@ -579,12 +579,15 @@ async function runErrorTestBody(mangledCompilations) {
 
   //we need the raw return data, and contract.call() does not exist yet,
   //so we're going to have to use web3.eth.call()
-
-  let data = await web3.eth.call({
-    to: deployedContract.address,
-    data: selector
-  });
-
+  let data;
+  try {
```

Comment:
Huh... I'm confused why this change is necessary even though VM errors are off.  What's going on here...? (@davidmurdoch?)

Reply:
See papertrail: https://github.com/trufflesuite/ganache/issues/1496

Reply:
We probably want this papertrail as a comment, actually! Leaving this unresolved until we get a comment :)

Reply:
@cds-amal please resolve when you feel this is sufficiently documented